docs/Reference/CLI-Syntax.md

```diff
@@ -0,0 +1,277 @@
+
+## Options
+
+### `allowedfutureblocktime`
+
+=== "Syntax"
+
+    ```bash
+    --allowedfutureblocktime <INTEGER>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --allowedfutureblocktime 2
+    ```
+
+Maximum time from current time, in seconds, allowed for blocks before they're considered future blocks.
+The default is 0.
+
+### `immutabilitythreshold`
+
+=== "Syntax"
+
+    ```bash
+    --immutabilitythreshold <INTEGER>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --immutabilitythreshold 1000000
+    ```
+
+Overrides the default immutability threshold for GoQuorum nodes.
+Blocks below the immutability threshold are moved to the `ancient` data folder.
+The default is 3162240.
+
+### `istanbul.blockperiod`
+
+=== "Syntax"
+
+    ```bash
+    --istanbul.blockperiod <INTEGER>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --istanbul.blockperiod 5
+    ```
+
+Minimum time between two consecutive [IBFT](../Concepts/Consensus/IBFT.md) blocks' timestamps in seconds.
+Setting the block period determines how quickly blocks should be minted by the validators.
+It is used for validation of block times by all nodes, so should not be changed after deciding a value for the network.
+
+The default is 1.
+
+### `istanbul.requesttimeout`
+
+=== "Syntax"
+
+    ```bash
+    --istanbul.requesttimeout <INTEGER>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --istanbul.requesttimeout 12000
+    ```
+
+Minimum request timeout for each [IBFT](../Concepts/Consensus/IBFT.md) round in milliseconds.
+The request timeout is the timeout at which IBFT triggers a new round if the previous one did not complete.
+This period increases as the timeout is hit more often.
+
+The default is 10000.
+
+### `multitenancy`
+
+=== "Syntax"
+
+    ```bash
+    --multitenancy
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --multitenancy
+    ```
+
+If included, [multi-tenancy](../Concepts/Multitenancy/Overview.md) is enabled.
+This requires the [JSON RPC Security plugin](../HowTo/Use/JSON-RPC-API-Security.md) to also be configured.
+
+### `permissioned`
+
+=== "Syntax"
+
+    ```bash
+    --permissioned
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --permissioned
+    ```
+
+If included, [basic network permissioning](../Concepts/Permissioning/BasicNetworkPermissions.md) is enabled.
+The node allows only a defined list of nodes to connect.
+
+### `plugins`
+
+=== "Syntax"
+
+    ```bash
+    --plugins file:///<path>/<to>/plugins.json
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --plugins file:///opt/geth/plugins.json
+    ```
+
+URI of the [plugins settings JSON file](../HowTo/Configure/Plugins.md).
+Use this to configure [plugins](../Concepts/Plugins/Plugins.md).
+
+### `plugins.localverify`
+
+=== "Syntax"
+
+    ```bash
+    --plugins.localverify
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --plugins.localverify
+    ```
+
+If included, [plugin integrity](../Concepts/Plugins/Plugins.md#plugin-integrity-verification) is verified from the
```

Comment:
You can probably remove "If included" from any descriptions. Seems redundant. Maybe just, "Verifies plugin integrity from..."